### ISSUE TITLE
Repeated indices bug

### ISSUE THREAD

**Issue by torhaugl** [MarcusTL12/ExcitationOperators.jl#13]
There is a problem with repeated indices. I'll highlight a two examples that should give the same result here.
```julia
using ExcitationOperators
using ExcitationOperators.BasicStuff.StandardIndices

exval(summation(E(k,j)*E(j,i), [i, j, k])) # 4 ∑_k()
exval(summation(E(i,j)*E(j,i), [i, j]))    # 4, but should be 4 ∑_i() or 4 ∑_j()
```

**Comment by torhaugl** [comment]
Doing summation outside exval works also

```julia
julia> exval(E(i,j)*E(j,i))
4 δ_ij

julia> summation(exval(E(i,j)*E(j,i)), [i,j])
4 ∑_j()
```

**Comment by torhaugl** [comment]
Fixed in PR #14
